src/types/genre.type.ts

```diff
@@ -0,0 +1,11 @@
+export enum Genre {
+  comedy = 'comedy',
```

Comment:
```suggestion
  Comedy = 'comedy',
```
ключи дожны быть с большой